build.fsx

```diff
@@ -291,6 +291,7 @@ let dotnetAssemblyInfos =
       "Fake.Testing.Common", "Common testing data types"
       "Fake.Tracing.NAntXml", "NAntXml"
       "Fake.Windows.Chocolatey", "Running and packaging with Chocolatey"
+      "Fake.Windows.Registry", "CRUD functionality for Windows registry"
```

Comment:
Can we add it to `help/templates/template.cshtml`, ie to add it to the menu in the `Windows` modules submenu?

Just reference the API-Reference there (no need to add a new markdown documentation)

Reply:
sorry you already added it.